coordinator/src/node/unrealized_pnl.rs

```diff
@@ -32,7 +34,7 @@ fn sync_position(
     position: &Position,
     quote: Quote,
 ) -> Result<()> {
-    let pnl = position.calculate_coordinator_pnl(quote)?;
+    let pnl = position.calculate_coordinator_pnl(quote, position.coordinator_leverage)?;
```

Comment:
:thinking: It's already in the `Position` that you are calling the method on! No need to pass it from the outside.

Reply:
good catch, thanks :)